### ISSUE TITLE
'Mantle.h' file not found

### ISSUE THREAD

**Issue by wellkunnected** [mitchellporter/Popstagram#1]
Trying to run this app and I am getting an error that the Mantle.H file is missing.

**Comment by mitchellporter** [comment]
The project uses Cocoapods to manage it's dependencies. You need to run `pod install` in terminal to install the pods, then the project will build and run.

**Comment by wellkunnected** [comment]
Yep, did that and the only error I get is that the "mantle.H file is
missing". For some reason I almost ALWAYS have an issue with any project
using cocoapods, they're a pain in my butt dude lol. Either a developer
releases a project that uses cocoapods and doesn't include an actual "pods"
folder in the project, so when running it in Xcode immediately before the
app is even built I can see the Pods file in red meaning it's not going to
build without the folder containing the files for the pod. In this case,
you guys included the pods folder but I'm getting an error regarding the
Mantle.H file which is trying to be imported via Xcode. All around
headache, because the best projects all use cocoapods :(

On Tue, Feb 9, 2016 at 3:27 PM, Mitchell Porter notifications@github.com
wrote:

> The project uses Cocoapods to manage it's dependencies. You need to run pod
> install in terminal to install the pods, then the project will build and
> run.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mitchellporter/Popstagram/issues/1#issuecomment-182046977
> .

## 

Founded Well Kunnected, Inc 2010
